main.tf

```diff
@@ -67,3 +67,16 @@ resource "azurerm_storage_account_network_rules" "netrule" {
   bypass                     = var.traffic_bypass
 }
 
+## azure reference https://docs.microsoft.com/en-us/azure/storage/common/infrastructure-encryption-enable?tabs=portal
+## Policy ADR: https://reedelsevier.sharepoint.com/sites/OG-CoP-Cloud/SitePages/Azure-Policy---Storage-Account.aspx
+resource "azurerm_storage_encryption_scope" "runner" {
+  count = var.enable_infrastructure_encryption ? 1 : 0
+
+  name                               = azurerm_storage_account.sa.name
+  storage_account_id                 = azurerm_storage_account.sa.id
+  source                             = "Microsoft.Storage"
+  infrastructure_encryption_required = true
```

Comment:
Should this resource always be called and the encryption required be toggled?
I know it defaults to false but at least it will be visible in the plan

Reply:
makes sense. I can do that